guides/manual-installation.md

```diff
@@ -419,7 +439,7 @@ certbot --nginx -d example.com -d www.example.com
 zypper install nginx
 ```
 
-**Step 2**: Add firewall exceptions for HTTP and HTTPS:
+**Step 2**: Add firewalld exceptions for HTTP and HTTPS **(ONLY if firewalld is enabled)**:
```

Comment:
@rtorrero has commented that to have docker engine running, firewalld must be running.
I don't know how accurate is that, but let's have an eye

Reply:
Docker engine not work without firewalld ? This is new to me :+1: 
@rtorrero maybe you can provide more info on that.

Reply:
It's some misconfiguration issue, docker itself does not have any dependencies on the firewall, it's some quirky iptables stuff I guess :/

Reply:
I installed Trento Server in a VM where firewalld was not running.

Reply:
The issue only appears when firewalld is uninstalled and not rebooted. I think it's safe to assume this is an edge case that can be ignored.